test_vm/src/lib.rs

```diff
@@ -990,6 +992,42 @@ impl<'invocation, 'bs> Runtime for InvocationCtx<'invocation, 'bs> {
         self.v.get_actor(Address::new_id(id)).and_then(|act| act.predictable_address)
     }
 
+    fn send(
+        &self,
+        to: &Address,
+        method: MethodNum,
+        params: Option<IpldBlock>,
+        value: TokenAmount,
+    ) -> Result<Option<IpldBlock>, ActorError> {
+        // TODO gas_limit is current ignored, what should we do about it?
```

Comment:
This TODO isn't helpful - there's nothing concretely to do or a condition upon which to do it. It's fine that gas limit is ignored, just leave a comment without TODO.